Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -188,18 +188,26 @@ open class Player: BaseObject {
 
     private func bindPlaybackEvents() {
         if let playback = core?.activePlayback {
```

Comment:
I know this is not from this PR, but we can change this if let to guard let, once we don't have code outside with a nullable core and we can remove one level of indentation.